pkg/plugins/manager/loader/loader_test.go

```diff
@@ -597,6 +597,116 @@ func TestLoader_Load_MultiplePlugins(t *testing.T) {
 	})
 }
 
+func TestLoader_Load_RBACReady(t *testing.T) {
+	pluginDir, err := filepath.Abs("../testdata/test-app-with-roles")
+	if err != nil {
+		t.Errorf("could not construct absolute path of current dir")
+		return
+	}
+
+	t.Run("Load with RBAC roles", func(t *testing.T) {
+		tests := []struct {
+			name            string
+			cfg             *config.Cfg
+			pluginPaths     []string
+			appURL          string
+			existingPlugins map[string]struct{}
+			want            []*plugins.Plugin
+			pluginErrors    map[string]*plugins.Error
```

Comment:
nit: `pluginErrors` doesn't seem to be specified in the test case, so can probably removed from the test and this struct.